packages/codemod/index.js

```diff
@@ -0,0 +1,20 @@
+import { addIntegrationToConfig } from './config/add.js';
+import { bgGreen, green, black, bgCyan, cyan } from 'kleur';
+
+async function run() {
+  const [command, ...args] = process.argv.splice(2);
+  if (command !== 'add') return;
+	if (!args.length) return;
+	for (const name of args) {
+		await addIntegrationToConfig({ name, package: `@astrojs/${name}` });
+		// TODO: add integration to devDependencies and add peerDependencies as dependencies
```

Comment:
wait, did you just accidentally build `astro add react`? 😆

Reply:
Because if you wanted to pull this out into an `astro add react` command I would not be mad at all. We'd talked about wanting that asap